.eslintrc.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+  env: {
+    browser: false,
+    node: true,
+    mocha: true,
+    es6: true
+  },
+  // add global vars for chai, etc
+  globals: {
+    expect: false,
+    chai: false,
+    sinon: false
+  },
+  rules: {
+    // possible errors
+    'no-extra-parens': 1,
+    'valid-jsdoc': [1, {
```

Comment:
what do you think about formatting the code like this?

```suggestion
    'valid-jsdoc': [
       1, 
       {
```